### PR TITLE
fix(deps): move devDependencies to dependencies

### PR DESCRIPTION
**Pull request by rtritto**
This change is [Reviewable](https://reviewable.io/reviews/mongo-express/mongo-express/1577)

### CODE REVIEW

**Review by xTudoS — APPROVED** [review]

**Comment by BlackthornYugen** [comment]
Why merge dev dependencies with runtime dependencies?

File: package.json [hunk 1]
```diff
@@ -52,11 +53,13 @@
     "free-swig": "^1.5.4",
     "gridfs-stream": "^1.1.1",
     "html-entities": "^2.5.2",
+    "jquery": "^3.7.1",
     "memorystore": "^1.6.7",
     "method-override": "^3.0.0",
     "mongodb": "^6.7.0",
     "mongodb-query-parser": "^4.1.3",
     "morgan": "^1.10.0",
+    "renderjson": "github:rtritto/renderjson#develop",
```
Comment (BlackthornYugen) [inline package.json:62]: Should this actually be a runtime dependency?

Comment (rtritto) [reply package.json:62]: Yes, it's used here [/lib/scripts/collection.js](https://github.com/mongo-express/mongo-express/blob/c937db575b5f7344e44d0a48a1acdeae6afd346a/lib/scripts/collection.js)

**Comment by rtritto** [comment]
> Why merge dev dependencies with runtime dependencies?

The dependencies that I moved are required (used) by main bundled app

**Pull request merged**